ahoopes16/helianthus
# Basic Navigation, Header, and Content Area

# Summary
If accepted, this PR will:
- Create the basic navigation bar for desktop users (no actual links yet)
- Create the initial header with logo for desktop users
- Create an initial content card area (with no real information)

# How to Test
1. Pull this branch and `yarn && yarn start`
2. Visit localhost:3000
3. Behold, it should look like this:
![image](https://user-images.githubusercontent.com/12129503/104968196-6d676a00-59a2-11eb-907c-46f8e3888fc5.png)
